src/app/shared/horizontal-carousel/horizontal-carousel.ts

```diff
@@ -0,0 +1,88 @@
+import {AfterViewInit, Component, NgModule, ViewEncapsulation} from '@angular/core';
+import {GuideItems} from '../guide-items/guide-items';
+import {MatCardModule} from '@angular/material/card';
+import {CommonModule} from '@angular/common';
+import {RouterModule} from '@angular/router';
+import {MatIconModule} from '@angular/material/icon';
+import {MatButtonModule} from '@angular/material/button';
+
+const CARD_WIDTH = 250;
+
+@Component({
+  selector: 'app-horizontal-carousel',
+  templateUrl: './horizontal-carousel.html',
+  styleUrls: ['./horizontal-carousel.scss'],
+  encapsulation: ViewEncapsulation.None,
```

Comment:
It looks like maybe you're using `viewEncapsulation.None` so the buttons are aligned correctly? It would be great if we could remove this.

What about having this component be `display: flex; flex-direction: row` and then having the buttons be hidden instead of removing them with `*ngIf`? That way they have a reserved space and the carousel won't jump when they show/hide.